sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates-client-side/README.md

```diff
@@ -0,0 +1,44 @@
+# Azure Spring Boot Key Vault Certificates client library for Java
+
+## Key concepts
+This sample illustrates how to use [Azure Spring Boot Starter Key Vault Certificates ][azure_spring_boot_starter_key_vault_certificates].
+
+This sample should work together with [azure-spring-boot-sample-keyvault-certificates-server-side].
+
+## Getting started
+
+### Prerequisites
```

Comment:
Let's use unified section captions and structures of all sample projects.
Prerequisites -> Environment checklist